src/LocalAuthentication/LAEnums.cs

```diff
@@ -13,8 +13,7 @@ public enum LAPolicy : nint {
 		DeviceOwnerAuthentication = 2
 	}
 
-	[iOS (8,0)]
-	[Availability (Platform.Mac_10_10)]
+	[iOS (8,0), Watch (3,0), TV (10,0), Mac (10,10)]
```

Comment:
The availability was added in the headers (looking at iOS or mac header)
```NS_ENUM_AVAILABLE(10_10, 8_0) __WATCHOS_AVAILABLE(3.0) __TVOS_AVAILABLE(10.0);```
Though the framework itself doesn't appear to be in the frameworks folder for tv/watch